Easydict/objc/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -406,6 +436,9 @@ - (void)startQueryText:(NSString *)text actionType:(EZActionType)actionType {
         return;
     }
     
+    // update tableView row
+    [self.tableView reloadData];
```

Comment:
Why call `reloadData` here?

Is it to handle some special case? I'm worried about this solution, it may complicate and confuse our code.

Reply:
yeah, I called `reloadData` to make the status of show tips update to the UI. I think your suggestion is right, I will try to fix it.

Reply:
How to fix this code?

Reply:
I have removed it.